grails-app/services/com/redhat/theses/config/Configuration.groovy

```diff
@@ -48,7 +48,7 @@ class Configuration {
         }
     }
 
-    private ConfigObject createDefaultConfig() {
+    public ConfigObject createDefaultConfig() {
```

Comment:
I need it in HomeController a ContactsController to get default values from default config if values are not yet present in Database to prevent migration issues like blank HomePage etc.